megatron/data/data_utils.py

```diff
@@ -74,49 +74,27 @@ def build_the_dataset(
         seq_length,
         seed
         ]
-    if neox_args.train_mtf:
-        dataset = DecoderPackedMTFDataset(
-            *dataset_args,
-            data_impl,
-            skip_warmup=False,
-            tokenizer=neox_args.tokenizer,
-        ) 
-    elif neox_args.training_objective == "mlm":
-
-        dataset = MLMDataset(
-            *dataset_args,
+
+    if neox_args.model_arch == "gpt2":
```

Comment:
Is there a particular reason to use GPT2 here over GPT3? I know that they're the same but I wouldn't necessarily assume all users know that.

Reply:
No strong reason, only done for the sake of naming consistency since it will call the model class object from gpt2_model.py

Reply:
lol I thought I had bumped those up to GPT-3 as well.. for now, this is the right choice (if only for consistency)